tools/tenscan/frontend/src/components/modules/dashboard/index.tsx

```diff
@@ -1,5 +1,4 @@
 import React from "react";
-import { CalendarDateRangePicker } from "@/src/components/date-range-picker";
 import {
```

Comment:
It's not shown here but the "Recent Blocks" on the homepage should be switched to "Recent Rollups" imo since the L1 blocks are sort of irrelevant to most people.